### DISCUSSION TITLE
C Completo e Total

### DISCUSSION THREAD

**Discussion by RodrigoDornelles** [pilhacheia/clube-do-livro-dev#15] [Books]
É um ótimo livro para aprender de verdade como funciona uma linguagem e o computador,
mesmo que você não trabalhe com C eu recomendo muito.

![c](https://user-images.githubusercontent.com/29123479/118137447-968d8980-b3db-11eb-88ad-85e434e8630d.png)

**Comment by lincolixavier** [comment]
Sensacional @RodrigoDornelles !

**Comment by ghost** [comment]
Um trabalho notável e que sempre leio, releio.  E que mudou minha vida ou me direcionou para o caminho da programação.
